drake-torch-melodic.dockerfile

```diff
@@ -123,6 +124,8 @@ RUN python3 -m pip install --upgrade --no-cache-dir --compile \
 
 RUN add-apt-repository -y ppa:git-core/ppa \
     && apt-get install -qy \
+        openssh-server \
```

Comment:
May be not for this PR, but we should pin the versions of the packages.

Reply:
With so many packages updated (we have to look at the logs to even know which one was updated), it's difficult to track down which update broke a build (if that happens). Do we really need to update everything? If not, then just select a few packages and update them, and let the tag reflect that (or attach metadata to the image using the `LABEL` keyword): https://www.balena.io/blog/our-dockerfile-tips-tricks/. 
As for doing it in an automated way, I dont have an answer now, let me think abt it a bit more, and get back to you. But I feel if things are not broken with the version already present (or we dont need an update to a version), should we even update them?

Reply:
I think we should. We've been plagued with issues from using old packages.